client/src/components/PastaShow.js

```diff
@@ -28,15 +34,67 @@ const PastaShow = (props) => {
     useEffect(() => {
         getPasta()
     }, [])
+
+    const postReview = async (newReviewData) => {
+        try {
+            const response = await fetch (`/api/v1/pastas/${pastaId}/newReview`, {
```

Comment:
Let's use proper RESTful pathing to post to `/api/v1/pastas/:id/reviews` rather than `newReviews`

https://restfulapi.net/http-methods/